Luminary099/R31.agc

```diff
@@ -125,7 +125,7 @@ REV83		CS	FLAGWRD7
 		SET
 			INTYPFLG	# CONIC EXTRAP.
 		STCALL	TET
-			INTEGRVS	# INTEGRATION --- AT LAST ---
+			INTEGRVS	# INTEGRATION --- AT LAST---
 OTHCONIC	VLOAD
 # Page 705
 			RATT
```

Comment:
3 lines below, no space between `EXTRAP.,` and `OTHER.`

![image](https://user-images.githubusercontent.com/3440094/95020751-ea293f00-0664-11eb-976c-d61e5e2bed46.png)